src/lib/pages/backoffice/Loan/LoanDetails/LoanActions/LoanActions.js

```diff
@@ -37,6 +37,8 @@ export default class LoanActions extends Component {
             />
           ) : (
             <Button
+              size="small"
+              fluid
```

Comment:
Can you please capitalize the actions text?
checkout -> Checkout ...
```
import capitalize from 'lodash/capitalize';


{capitalize(action)}
```
we could also simply use CSS, we would have to change it everywhere